docs/content/concepts/partitions-schedules-sensors/backfills.mdx

```diff
@@ -58,7 +58,14 @@ To get this behavior, you need to:
 
   Which property to use depends on whether it's most convenient for you to operate on start/end datetime objects, start/end partition keys, or a list of partition keys.
 
-For example:
+**If you're using an [I/O manager](/concepts/io-management/io-managers)**, you'll also need to make sure that the I/O manager is using these methods. **Note**: With the exception of file system I/O managers, Dagster's built-in database I/O managers - like [Snowflake](/integrations/snowflake), [BigQuery](/integrations/bigquery), or [DuckDB](/\_apidocs/libraries/dagster-duckdb) - include this functionality out of the box.
```

Comment:
```suggestion
**If you're using an [I/O manager](/concepts/io-management/io-managers)**, you'll also need to make sure that the I/O manager is using these methods. **Note**: File system I/O managers do not support single-run backfills, but Dagster's built-in database I/O managers - like [Snowflake](/integrations/snowflake), [BigQuery](/integrations/bigquery), or [DuckDB](/\_apidocs/libraries/dagster-duckdb) - include this functionality out of the box.
```
nit - this just reads a bit better to me, but not a necessary change

Reply:
This is better, thank you!